Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:centos7
+LABEL maintainer="Nik Hodgkinson <11xor6@gmail.com>"
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.11.0
+RUN yum -y install epel-release && \
+    yum -y update && \
+    yum -y install java-1.11.0-openjdk-devel python-pip && \
```

Comment:
Maybe we rely on more conservative Java version, so that such docker container could be used in K8S production?
Which version do you think we support @martint ?

Reply:
Happy to go either way here, just need a decision.

Reply:
We ran Java 11 in production and would like to move development to that version. I don’t see any reason to use a Java 8 when we have control over the version.

Reply:
Somewhat on the topic of Java version, @ChethanUK on Slack mentioned using the official OpenJDK docker images as a base, instead of installing it ourselves. I'm inclined to go in that direction, but I'd like some feedback on that particular path. Additionally the choice of which version to use is also important and probably needs to be resolved first.

Reply:
This version will use Java 11 installed via the OS package as is currently the case. We can revisit this at a later date if necessary.